### PR TITLE
UCT/DC: Add wakeup support for DC.

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3997/ for details (Mellanox internal link).

File: src/uct/ib/dc/accel/dc_mlx5.c [hunk 1]
```diff
@@ -96,30 +96,6 @@ static ucs_status_t uct_dc_mlx5_iface_query(uct_iface_h tl_iface, uct_iface_attr
...
-                                      UCT_IFACE_FLAG_ATOMIC_ADD32|
-                                      UCT_IFACE_FLAG_ATOMIC_FADD32|
-                                      UCT_IFACE_FLAG_ATOMIC_SWAP32|
-                                      UCT_IFACE_FLAG_ATOMIC_CSWAP32|
-                                      UCT_IFACE_FLAG_ATOMIC_DEVICE |
-                                      UCT_IFACE_FLAG_PENDING|
-                                      UCT_IFACE_FLAG_AM_CB_SYNC|
-                                      UCT_IFACE_FLAG_CONNECT_TO_IFACE|
-                                      UCT_IFACE_FLAG_ERRHANDLE_PEER_FAILURE;
-
```
Comment (shamisp) [inline src/uct/ib/dc/accel/dc_mlx5.c]: did we define 0 and all supported ?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3998/ for details (Mellanox internal link).

File: src/uct/ib/dc/accel/dc_mlx5.c (hunk 1 above)
Comment (yosefe) [inline src/uct/ib/dc/accel/dc_mlx5.c]: the flags are already filled by uct_dc_iface_query and uct_rc_mlx5_iface_common_query

**Pull request merged**